1.1.0/jessie/build-projectjson/README.md

```diff
@@ -11,8 +11,10 @@ This repository contains images that are used to compile/publish ASP.NET Core ap
 
 - [`1.1.0-projectjson`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1.0/jessie/build-projectjson/Dockerfile)
```

Comment:
I didn't have a strong reason for it, I just did what seemed to make sense without spending any time looking at others ordering.